.github/workflows/release_step_1_create_version_bump.yml

```diff
@@ -39,8 +39,9 @@ jobs:
         git config --global user.name "github-actions[bot]"
         bundle exec fastlane bump bump_type:${{ github.event.inputs.bump_type }}
       env:
-        GITHUB_USER_NAME: fastlane
-        GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        GITHUB_USER_NAME: fastlane # Todo: This is needed for docs - remove somehow
+        GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }} # Todo: This is needed for docs - remove somehow
```

Comment:
FYI, there were no bumps for `fastlane/docs` repo beyond 2.220.0, even though the latest had docs changes so there would be a diff to commit/PR. Is it somehow possible that these new bump lanes now avoid the release lane, which is the only place to run :update_docs and it's not being pushed now?

Reply:
@rogerluan FYI, no newer docs getting published. Wanna take a look at the lanes setup here now pls?

Reply:
What do you mean exactly: the new release automation here is not opening the docs update PR in the docs repo? Or it's not automatically publishing a docs update in the docs repo? Because it never did the latter AFAIK, only the former

Reply:
Neither. There was no PR open for the past few versions / months.

_I'm trying to trace the Fastfile lanes to when they're being called ([`:release` to `:update_docs`](https://github.com/fastlane/fastlane/blob/2c34057215aeabdaac4aea7806a8f166417c8dff/fastlane/Fastfile#L333)) but with not much luck, it only seems as the new bump logic somehow avoids these calls?_